index.html

```diff
@@ -559,6 +559,22 @@ <h3>Use Cases and Requirements</h3>
 or <a>verifiers</a>, because the conformance of ecosystem roles are highly
 application, use case, and market vertical specific.
         </p>
+
+        <p>
+While digital proofs, a subset of which are digital signatures, are required
```

Comment:
This could be worded to also explain that processing of the proof may be dependent on the syntax, such as using the JWS of a JWT for proofing a key holder. Some of the text of #539 tries to address this.

Reply:
Fixed in 56a7b6b382369b68f345ec8c854ed64c12b91f3f.